src/fun/builtins.bend

```diff
@@ -316,8 +316,13 @@ def IO/FS/read_line.read_chunks(fd, chunks):
         return wrap(bytes)
       case Result/Err:
```

Comment:
Can you add a comment explaining the meaning of each branch? It's getting hard to read
(ok means found new line, backtrack and join chunks, err means no new line, length == 0 means file over, so return as if new line, else means the line is still ongoing